tutorials/plot_compute_evoked_inverse_solution.py

```diff
@@ -43,15 +43,14 @@
 src_outputs = hcp.anatomy.compute_forward_stack(
     subject=subject, subjects_dir=subjects_dir,
     hcp_path=hcp_path, recordings_path=recordings_path,
-    # speed up computations here. Setting `add_dist` to True may improve the
-    # accuracy.
+    # Speed up computations here by setting `add_dist` to False.
```

Comment:
The `add_dists` also adds patch info which is used in forward computation to make things slightly better